packages/table/src/cell/cell.tsx

```diff
@@ -47,7 +47,7 @@ export interface ICellProps extends IIntentProps, IProps {
 
 export type ICellRenderer = (rowIndex: number, columnIndex: number) => React.ReactElement<ICellProps>;
 
-export const emptyCellRenderer = (_rowIndex: number, _columnIndex: number) => <Cell />;
+export const emptyCellRenderer: ICellRenderer = (_rowIndex: number, _columnIndex: number) => <Cell />;
```

Comment:
this change shouldn't be necessary. you can even remove the args since they're not used. a function with fewer args will pass type checking.